docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -245,16 +255,71 @@ slow to return results.
 (such as AWS Athena and BigQuery). Repeatedly querying for this data can easily
 rack up costs.
 
+## Optimizing Pre-Aggregation Build Times
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| For ideal performance, pre-aggregations should be built using a dedicated
+| Refresh Worker. [See here for more details][ref-prod-list-refresh].
+<!-- prettier-ignore-end -->
+
+By default, Cube.js will use the source database as a temporary staging area for
+writing pre-aggregations before moving them to Cube Store.
+
+> Get more info about default strategy
+
+### Batching
+
+Batching is a slightly more performant strategy where Cube.js sends compressed
+CSVs for Cube Store to ingest.
+
+![](build-batching.png)
+
+The performance scales to the amount of memory available on the Cube.js
+instance. Support is currently available for:
+
+- [AWS Athena][ref-connect-db-athena] (coming soon)
+- [AWS Redshift][ref-connect-db-redshift]
+- [BigQuery][ref-connect-db-bigquery]
+- [MySQL][ref-connect-db-mysql]
+- [Postgres][ref-connect-db-postgres]
+
+### Export bucket
+
+When dealing with larger pre-aggregations (more than 500k rows), performance can
```

Comment:
Right now, the threshold is 100k row, after that, you will get a warning that it's recommended to use export:

```suggestion
When dealing with larger pre-aggregations (more than 100k rows), performance can
```